{{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/ext.py

```diff
@@ -5,6 +5,7 @@
 
 from flask_babelex import gettext as _
 
+from . import congig
```

Comment:
there's a small typo here ``congig`` -> ``config``. When you fix it, can you also rebase on top of latest master